manifests/certificate/x509.pp

```diff
@@ -115,7 +131,7 @@
   $_csr = pick($csr, "${_csr_dir}/${name}.csr")
   $_key = pick($key, "${_key_dir}/${name}.key")
 
-  if !empty($altnames) {
+  if (!empty($altnames)) or (!empty($extkeyusage)) {
```

Comment:
or maybe `!empty($altnames+$extkeyusage)` would work?

Reply:
It does work. Changed as advised, thanks.